extension_helpers/__init__.py

```diff
@@ -2,3 +2,17 @@
 from ._setup_helpers import get_compiler, get_extensions, pkg_config
 from ._utils import import_file, write_if_different
 from .version import version as __version__
+
+from configparser import ConfigParser
+
+
+def _finalize_distribution_hook(distribution):
+    """
+    Something something setuptools entrypoint
+    """
+    config_files = distribution.find_config_files()
+    cfg = ConfigParser()
```

Comment:
```suggestion
    if len(config_files) == 0:
        return
    cfg = ConfigParser()
```

Sometimes, as in the failing test, there is no config file and we should assume extension helpers should not be auto-used.